parachains/runtimes/assets/statemint/src/xcm_config.rs

```diff
@@ -378,3 +447,18 @@ impl cumulus_pallet_xcm::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
 	type XcmExecutor = XcmExecutor<XcmConfig>;
 }
+
+pub type ForeignCreatorsSovereignAccountOf = (
+	SiblingParachainConvertsVia<Sibling, AccountId>,
+	AccountId32Aliases<RelayNetwork, AccountId>,
+	ParentIsPreset<AccountId>,
```

Comment:
the type used to convert the origin of foreign creator to account id. 
every foreign asset must start with (1, Parachain(...), ...), and it's creator (defined by `IsForeign` of `ForeignCreators`). 
so I do not see the case for these two implementations of `Covert`, they are never applicable in current setup.

Reply:
You are saying this is equivalent? My inclination is to leave it since it's been audited and is on Statemine/Westmint. But we could change this separately (in all three runtimes) as a follow up. Or do you think it should be removed now?
```rust
pub type ForeignCreatorsSovereignAccountOf = (
	SiblingParachainConvertsVia<Sibling, AccountId>,
);
```